src/convertToObject.js

```diff
@@ -7,6 +7,24 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const stylesLines = sourceString.split(';').map((item) => item.trim());
```

Comment:
There is no need to map over the array just to trim the items, as you can trim them later when you actually use them. This would improve performance by avoiding an unnecessary iteration over the array.